src/scripts/main.js

```diff
@@ -1,3 +1,33 @@
 'use strict';
 
-// write code here
+let employees = Array.from(document.querySelectorAll('ul li'));
+
+function sortList(list) {
+  employees = list.sort((a, b) => {
```

Comment:
The `sortList` function is modifying the global `employees` variable. It's better to avoid modifying global variables within functions to prevent unexpected side effects. Consider returning the sorted list and assigning it outside the function.